python/packages/sdk/serverless_sdk/sdk/base.py

```diff
@@ -15,19 +16,28 @@
 ]
 
 
+class TraceSpans(SimpleNamespace):
+    @property
+    def root(self):
+        return trace.root_span
+
+
 class ServerlessSdk:
     name: Final[str] = __name__
     version: Final[str] = __version__
 
-    trace_spans: Final = ...
+    trace_spans: TraceSpans
     instrumentation: Final = ...
```

Comment:
Just wondering, what this `Final = ...` notation means? (looks as syntax error)

Reply:
That's like a placeholder, we'll change it when we actually implement `serverlessSdk.instrumentation`. It's not a syntax error in my environment, maybe you have an old Python version?

Reply:
Got it, so it returns an empty object like a stub:
```
>>> from serverless_sdk import serverlessSdk
>>> serverlessSdk.instrumentation
Ellipsis
```